src/routes/e2e/widget/+page.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
```

Comment:
I added this file as a "page" that has no widget, just simple user info request. I think I'll add a bit more here, but the general intent is to have a route that doesn't have the widget, but still doesn't use the underlying SDK. This tests the configuration feature.